yarn.lock

```diff
@@ -12495,7 +12514,7 @@ uuid@^3.0.1, uuid@^3.3.2:
     eslint-plugin-react "^7.12.4"
     eslint-plugin-react-hooks "^1.6.0"
     eslint-plugin-scanjs-rules "^0.2.1"
-    eslint-plugin-va-enzyme "file:./packages/vagov-eslint/custom-eslint/va-enzyme"
+    eslint-plugin-va-enzyme "file:../../../Library/Caches/Yarn/v4/npm-vagov-eslint-1.0.0-e767bb3b-4bb9-4325-bca9-089aeb09942d-1598032953380/node_modules/vagov-eslint/custom-eslint/va-enzyme"
```

Comment:
This change looks unusual to me. Did you make this change manually, or was this a generated change?

Reply:
@billfienberg it was a generated change.